save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FileUploader.kt

```diff
@@ -224,56 +274,67 @@ private fun fileUploader() = FC<UploaderProps> { props ->
                             disabled = true
                             +"Select a file from existing"
                         }
-                        props.availableFiles.sortedByDescending { it.key.uploadedMillis }.map {
+
+                        availableFiles.sortedByDescending {
+                            if (it is FileInfo) {
+                                it.key.uploadedMillis
+                            } else {
+                                it.sizeBytes
```

Comment:
I think we should sort by name if `uploadedMillis` is not available

Reply:
actually, suggestion to sort by `toPrettyString` for both cases

Reply:
Done